openapi/openapi.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 info:
   title: BringAuto Fleet Management v2 API
   description: Specification for BringAuto fleet backend HTTP API
-  version: 2.0.0
+  version: 2.0.2
```

Comment:
Maybe 2.1.0? This behav. was not intended for the beginning of the implementation so it is a new feature in general.